power_v10/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_shared::error::ExitCode;
+use fvm_shared::METHOD_CONSTRUCTOR;
+use num_derive::FromPrimitive;
+
+pub use self::policy::*;
+pub use self::state::*;
+pub use self::types::*;
+
+#[doc(hidden)]
+pub mod ext;
+mod policy;
+mod state;
+pub mod testing;
+mod types;
+
+// * Updated to specs-actors commit: 999e57a151cc7ada020ca2844b651499ab8c0dec (v3.0.1)
+
+/// GasOnSubmitVerifySeal is amount of gas charged for SubmitPoRepForBulkVerify
+/// This number is empirically determined
+pub mod detail {
+    pub const GAS_ON_SUBMIT_VERIFY_SEAL: i64 = 34721049;
+}
+
+/// Storage power actor methods available
+#[derive(FromPrimitive)]
+#[repr(u64)]
+pub enum Method {
+    /// Constructor for Storage Power Actor
+    Constructor = METHOD_CONSTRUCTOR,
+    CreateMiner = 2,
+    UpdateClaimedPower = 3,
+    EnrollCronEvent = 4,
+    OnEpochTickEnd = 5,
+    UpdatePledgeTotal = 6,
+    // * Deprecated in v2
+    // OnConsensusFault = 7,
+    SubmitPoRepForBulkVerify = 8,
+    CurrentTotalPower = 9,
+    // Method numbers derived from FRC-0042 standards
+    CreateMinerExported = frc42_dispatch::method_hash!("CreateMiner"),
+    NetworkRawPowerExported = frc42_dispatch::method_hash!("NetworkRawPower"),
+    MinerRawPowerExported = frc42_dispatch::method_hash!("MinerRawPower"),
+    MinerCountExported = frc42_dispatch::method_hash!("MinerCount"),
+    MinerConsensusCountExported = frc42_dispatch::method_hash!("MinerConsensusCount"),
```

Comment:
the (current) builtin-actors repo do not have the above 5 variants these, curious as to why we've added it here?

Reply:
those are there, please ignore